File: p_prodsec/2nd_tarea/fibo2nd.go
package main

import (
	"fmt"
)

func main() {
	x := 0
	y := 1
	fibo(x, y)
}

func fibo(x, y int) {
	var z int8
	fmt.Print("inserte el numero de iteraciones: ")
	fmt.Scanln(&z)
	fmt.Println(z)
	fibo_calc(x, y, z)
}
func fibo_calc(x, y int, z int8) {
	var xy int
	for i := 0; i < int(z); i++ {
		fmt.Println(x)
		xy = x
		x = y
		y = xy + x
	}
}

File: p_prodsec/2nd_tarea/fibo2nd.py
def main():
    x=0
    y=1
    fibo(x,y)

def fibo(x,y):
    while True:
        try:
            z= int(input("introduce un numero: "))
            break
        except ValueError:
            print("El valor introducido no es un numero")

    fibo_calc(x,y,z)

def fibo_calc(x,y,z):
    for i in range(z):
        print(x)
        xy = x
        x = y
        y = xy +  x

main()